Community/alumni.md

```diff
@@ -2,33 +2,61 @@
-|[Pranjal Aswani](#pranjal-aswani)|2018|Reporting of CHAOSS Metrics: Refactoring the existing code and extending the capabilities of the Manuscripts Project|Valerio Cosentino, Jesus Gonzalez-Barahona|
+|Student|Year|Program|Project|Mentors|
+|-------|----|-------|-------|-------|
+|[Jaskirat Singh](#Jaskirat-Singh)|2020|GSoD|Create a CHAOSS Community-wide Handbook|Georg J.P.link, Armstrong Foundjem, Matt Germonprez
+|[Xiaoya Esther](#Xiaoya-Esther)|2020|GSoD|Build documentation for CHAOSS D&I Badging project|Matt Snell, Aastha Bist
+|[Ria Gupta](#Ria-Gupta)|2020|GSoC|Social Currency Metric System|Valerio Cosentino, Samantha Venia, Logan|
+|[Tianyi Zhou](#Tianyi-Zhou)|2020|GSoC|Large Social Network Analysis and Anomaly Detection with Augur|Sean Goggins, Jonahz, Gabe Heim|
+|[Sarit Adhikari](#Sarit-Adhikari)|2020|GSoC|Machine Learning for Anomaly Detection in Open Source Communities|Sean Goggins, Carter Landis, Gabe Heim|
+|[Abhinav Bajpai](#Abhinav-Bajpai)|2020|GSoC|Implementing GitLab Data Collection Worker and Mapper to bind the responses of GitLab API, Github API & the Augur schema|Sean Goggins, Carter Landis|
+|[bistaastha](#bistaastha)|2020|GSoC|Build Workflow process for CHAOSS D&I Badging Project|Matt Snell|
+|[vchrombie](#vchrombie)|2020|GSoC|Creating Quality models using GrimoireLab and CHAOSS metrics|Valerio Cosentino, Aniruddha Karajgi|
+|[Saicharan Reddy](#Saicharan-Reddy)|2020|GSoC|Implementation of GitLab Data Collection Worker & Test Coverage Improvement|Elita Nelson, Sean Goggins, Jonahz|
+|[Akshara P](#Akshara-P)|2020|GSoC|Machine Learning for Anomaly Detection in Open Source Communities|Elita Nelson, Sean Goggins, Gabe Heim|
+|[Pratik Mishra](#Pratik-Mishra)|2020|GSoC|Machine Learning for Anomaly Detection in Open Source Communities|Elita Nelson, Sean Goggins, Gabe Heim|
+|[Ore-Aruwaji Oloruntola](#Ore-Aruwaji-Oloruntola)|2020|Outreachy|Build Workflow Process for CHAOSS Diversity & Inclusion Badging|Matt Germonprez, Matt Snell, Saleh Abdel Motaal|
+|[Parth Sharma](#parth-sharma)|2019|GSoC|Build CHAOSS Risk and Growth Maturity and Decline Metrics in Augur|Sean Goggins|
+|[Bingwen Ma](#bingwen-ma)|2019|GSoC|Build CHAOSS Risk and Growth Maturity and Decline Metrics in Augur|Sean Goggins, Matt Germonprez|
+|[Aniruddha Karajgi](#aniruddha-karajgi)|2019|GSoC|Implementing CHAOSS Metrics with Perceval|Jesus Gonzalez-Barahona, valcos, Pranjal Aswani|
+|[Nishchith K Shetty](#nishchith-k-shetty)|2019|GSoC|Support of Source Code Related Metrics|Jesus Gonzalez-Barahona, valcos, Pranjal Aswani|
+|[Keanu Nichols](#keanu-nichols)|2018|GSoC|Reporting of CHAOSS Metrics|Sean Goggins, Jesus Gonzalez-Barahona|
+|[Pranjal Aswani](#pranjal-aswani)|2018|GSoC|Reporting of CHAOSS Metrics: Refactoring the existing code and extending the capabilities of the Manuscripts Project|Valerio Cosentino, Jesus Gonzalez-Barahona|
 
 
 ## Mentees
 
+### Jaskirat Singh
+
+Jaskirat Singh selected for GSoD 2020

```

Comment:
How about mentioning the Linux Foundation here?
`Jaskirat Singh was selected for GSoD 2020 under the Linux Foundation.`

WDYT @klumb?